scripts/noshake.json

```diff
@@ -327,6 +327,8 @@
   "Mathlib.Tactic.ArithMult": ["Mathlib.Tactic.ArithMult.Init"],
   "Mathlib.Tactic.Algebraize": ["Mathlib.Algebra.Algebra.Tower"],
   "Mathlib.RingTheory.SimpleModule.Basic": ["Mathlib.Data.Matrix.Mul"],
+  "Mathlib.RingTheory.PowerSeries.Evaluation":
```

Comment:
What is the problem with these files?

Reply:
It requires to import `Polynomial`, because otherwise the namespace doesn't exist.

Reply:
Is there a smaller import? `PiTopology` doesn't seem like the one you want for `Polynomial`.

Reply:
I think perhaps you meant `WithPiTopology` not `Polynomial`?

Reply:
Either way, I deleted this line and everything was fine.